PipelinesTasks/PRMetrics/buildAndReleaseTask/invokers/azureReposInvoker.ts

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IGitApi } from "azure-devops-node-api/gitApi";
+import { JsonPatchDocument, JsonPatchOperation, Operation } from "azure-devops-node-api/interfaces/common/VSSInterfaces";
+import { Comment, CommentThreadStatus, GitPullRequest, GitPullRequestCommentThread, GitPullRequestIteration } from "azure-devops-node-api/interfaces/GitInterfaces";
+import DevOpsApiWrapper from "../wrappers/devOpsApiWrapper";
+import TaskLibWrapper from "../wrappers/taskLibWrapper";
+
+export default class AzureReposInvoker {
+    private taskLibWrapper: TaskLibWrapper;
+    private devOpsApiWrapper: DevOpsApiWrapper;
+    private gitApi: IGitApi | undefined;
+
+    private baseUri = `${process.env["SYSTEM_TEAMFOUNDATIONCOLLECTIONURI"]}`;
+    private project = process.env["SYSTEM_TEAMPROJECT"];
+    private repositoryId = process.env["BUILD_REPOSITORY_ID"] as string;
+    private pullRequestId = process.env["SYSTEM_PULLREQUEST_PULLREQUESTID"] ? parseInt(process.env["SYSTEM_PULLREQUEST_PULLREQUESTID"]) : 0;
+    private azurePAT = process.env["SYSTEM_ACCESSTOKEN"];
+
+    /**
+      * Initializes a new instance of the AzureReposInvoker class.
+      * @param devOpsApiWrapper The wrapper around the Azure Devops Api Task Lib.
+      * @param taskLibWrapper The wrapper around the Azure Pipelines Task Lib.
+      */
+    public constructor(devOpsApiWrapper: DevOpsApiWrapper, taskLibWrapper: TaskLibWrapper) {
+        this.devOpsApiWrapper = devOpsApiWrapper;
+        this.taskLibWrapper = taskLibWrapper;
+    }
+
+    /**
+      * Gets the pull request from the devops api.
+      */
+    public getDetails = async (): Promise<GitPullRequest> => {
```

Comment:
I only need title and description here

Reply:
Created a new interface and returned only those.